dl-common/src/test/java/com/ucar/datalink/common/utils/StrUtilTest.java

```diff
@@ -0,0 +1,30 @@
+package com.ucar.datalink.common.utils;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+import static com.ucar.datalink.common.utils.StrUtil.*;
+
+public class StrUtilTest {
+
+    @Test
+    public void testStringify() {
+        assertEquals("1B", stringify(1));
+        assertEquals("1.00KB", stringify(1024));
+        assertEquals("1.00MB", stringify(1048576));
+        assertEquals("1.00GB", stringify(1073741824));
+        assertEquals("1.00GB", stringify(1073741824));
```

Comment:
Duplicate assert. Lines 15 and 16 are identical.

Reply:
Thank you